### PR TITLE
Add preliminary list of blue button report mappings

### PR DESCRIPTION
**Pull request by bshyong**
Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/963

List of ambiguities here that we will resolve later this week with MHV:
- missing MHV key for `Labs and Tests, Self-Reported`
- missing MHV key for `Vitals and Readings, Self-Reported`
- what is the difference between `prescription` and `medication`?
- what does `vachemlab` map to?
- what does `vahth` map to?

### CODE REVIEW

File: src/js/blue-button/config.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        label: 'VA Electrocardiogram (EKG) History performed at VA Treating Facilities',
+      },
+      {
+        value: '',
+        label: 'Labs and Tests, Self-Reported',
+      },
+    ],
+    ehr: [
+      {
+        value: 'vaproblemlis',
```
Comment (U-DON) [inline src/js/blue-button/config.js]: `vaproblemlist`

File: src/js/blue-button/config.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+      },
+    ],
+    labs: [
+      {
+        value: 'labsandtest',
+        label: 'VA Laboratory Results',
+        hold: 3,
+      },
+      {
+        value: 'vapatholog',
```
Comment (U-DON) [inline src/js/blue-button/config.js]: `vapathology`

**Review by U-DON — CHANGES_REQUESTED** [review]
Aside from the ones I commented on, it looks like all of the values are missing their last characters.

**Comment by U-DON** [comment]
Might also consider giving each report type a title too.

File: src/js/blue-button/config.js [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+        label: 'VA Pathology Reports',
+        hold: 14,
+      },
+      {
+        value: 'varadiolog',
```
Comment (austin-martinez) [inline src/js/blue-button/config.js]: `varadiology`
Looks like a lot of these are missing the last letter
`welnes` -> `wellness`, etc.

**Comment by bshyong** [comment]
@U-DON @austin-martinez I believe the key names are intentional - they depend on what MHV will take; I pulled them from the blue button client in the API https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/bb/generate_report_request_form.rb#L6

**Comment by U-DON** [comment]
@bshyong It seems like the keys in the code you linked have their last letters, though. Like the first one, `seiactivityjournal` has that last `l`, but the one in the config is `seiactivityjourna` without the `l`.

**Comment by bshyong** [comment]
Adding titles is a good suggestion; we might as well have it all in the config file. Those are added in the latest commit. Key names are also fixed

**Review by U-DON — APPROVED** [review]
LGTM.

**Pull request merged**